managed/services/telemetry/config.default.yml

```diff
@@ -937,35 +937,28 @@ telemetry:
       - metric_name: "postgresql_db_count"
         value: 1
 
-  # Note: use these for testing and PMM-12462
-  # - id: PMMServerFeatureToggles
-  #   source: ENV_VARS
-  #   summary: "Use of feature toggles in PMM Server"
-  #   data:
-  #     - metric_name: "pmm_server_disable_telemetry"
-  #       column: "DISABLE_TELEMETRY"
-  #     - metric_name: "pmm_server_enable_alerting"
-  #       column: "ENABLE_ALERTING"
-  #     - metric_name: "pmm_server_enable_backup_management"
-  #       column: "ENABLE_BACKUP_MANAGEMENT"
-  #     - metric_name: "pmm_server_enable_debug"
-  #       column: "ENABLE_DEBUG"
-  #     - metric_name: "pmm_server_enable_rbac"
-  #       column: "ENABLE_RBAC"
-
-  # - id: PMMServerFeatureTogglesStripValues
-  #   source: ENV_VARS
-  #   summary: "Use of feature toggles in PMM Server"
-  #   transform:
-  #     type: StripValues
-  #   data:
-  #     - metric_name: "pmm_server_disable_telemetry"
-  #       column: "DISABLE_TELEMETRY"
-  #     - metric_name: "pmm_server_enable_alerting"
-  #       column: "ENABLE_ALERTING"
-  #     - metric_name: "pmm_server_enable_backup_management"
-  #       column: "ENABLE_BACKUP_MANAGEMENT"
-  #     - metric_name: "pmm_server_enable_debug"
-  #       column: "ENABLE_DEBUG"
-  #     - metric_name: "pmm_server_enable_rbac"
-  #       column: "ENABLE_RBAC"
+  - id: PMMServerHA
+    source: ENV_VARS
+    summary: "Use of HA feature"
+    data:
+      - metric_name: "pmm_server_disable_builtin_clickhouse"
+        column: "PERCONA_TEST_PMM_DISABLE_BUILTIN_CLICKHOUSE"
+      - metric_name: "pmm_server_disable_builtin_postgres"
+        column: "PERCONA_TEST_PMM_DISABLE_BUILTIN_POSTGRES"
+      - metric_name: "pmm_server_ha_enable"
+        column: "PERCONA_TEST_HA_ENABLE"
+      - metric_name: "pmm_server_ha_bootstrap"
+        column: "PERCONA_TEST_HA_BOOTSTRAP"
```

Comment:
I think we need only PERCONA_TEST_HA_ENABLE here

Reply:
OK, done!